cli/docker/app/factory.go

```diff
@@ -8,10 +8,12 @@ import (
 	"github.com/docker/libcompose/project"
 )
 
-type ProjectFactory struct {
+// DockerProjectFactory is a struct that hold the app.ProjectFactory implementation.
+type DockerProjectFactory struct {
```

Comment:
It's pretty common in go to do this because any time you reference the type you also include package.  So calling code would look like `dockerApp.ProjectFactory`.   Making the type have the package name in it then makes it redundant `dockerApp.DockerProjectFactory`.